setup.py

```diff
@@ -47,13 +47,13 @@
             'black<=19.10b0',
             'flake8<=3.9',
             'isort~=5.0',
-            'readme-renderer~=26.0',
+            'readme-renderer~=28.0',
```

Comment:
``suggestion
            'readme-renderer~=28.0.0',
```
`readme-renderer~=28.0.` translates to `readme-renderer==28.*`. This has created problems in openwisp-notifications earlier. Please check [compatible release in PEP 440](https://www.python.org/dev/peps/pep-0440/#compatible-release) for more info.

Reply:
The problem is not everyone knows about / follows [semver.org](https://semver.org/). This particular project does not follow having a minor version, they only have minor fixes and major releases, [check their changelog](https://github.com/pypa/readme_renderer/blob/master/CHANGES.rst).
Hence, `readme-renderer==28.*` would cover all the minor bug fixes in the `28` version, with `readme-renderer~=28.0.0` we would lose that, what do you think? :smile:

Reply:
I had a bad experience with this which broke dependencies of openwisp-notifications thrice. :sweat_smile: This was my only concern for being cautious. Thanks for sharing changelog of the module. I think we can keep `readme-renderer~=28.0`.

Reply:
Looks like they don't follow semver so what @atb00ker did should be correct. Do you agree @pandafy or do you have additional concerns?

Reply:
What @atb00ker has done is correct. :smile: